groovy/material/FirstDegreeMaterials.groovy

```diff
@@ -162,6 +162,13 @@ public class FirstDegreeMaterials {
                 .colorAverage()
                 .build();
 
+        CobaltMatte = new Material.Builder(8106, SuSyUtility.susyId("cobalt_matte"))
+                .ingot()
+                .flags(NO_UNIFICATION)
+                .color(0x7070d6)
+                .build()
+                .setFormula("(Co)(?)")
```

Comment:
you forgot to add `true` statement after the formula

Reply:
gocha